packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -132,40 +168,73 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 			statusComponents.push(connectionStateToString(containerState.connectionState));
 		}
 	}
-	const statusString = statusComponents.join(" | ");
+
+	function ContainerIDRow(label: string, id: string | undefined): React.ReactElement {
+		return (
+			<TableRow>
+				<TableCell {...columnSizing_unstable.getTableCellProps("containerProperty")}>
+					<b>{label}</b>
+				</TableCell>
+				<TableCell>{id}</TableCell>
+			</TableRow>
+		);
+	}
+
+	function ContainerStatusRow(): React.ReactElement {
+		return (
+			<TableRow>
+				<TableCell>
+					<b>Status</b>
+				</TableCell>
+				<TableCell>
+					<TableCellLayout
+						media={((): JSX.Element => {
+							switch (statusComponents[0]) {
+								case "attaching":
+									return (
+										<Badge shape="rounded" color="warning">
+											{statusComponents[0]}
+										</Badge>
+									);
+								case "detached":
+									return (
+										<Badge shape="rounded" color="danger">
+											{statusComponents[0]}
+										</Badge>
+									);
+								default:
+									return (
+										<Badge shape="rounded" color="success">
+											{statusComponents[0]}
+										</Badge>
+									);
+							}
+						})()}
+					>
+						{statusComponents[1] === "Connected" ? (
+							<Badge shape="rounded" color="success">
+								{statusComponents[1]}
+							</Badge>
+						) : (
+							<Badge shape="rounded" color="danger">
+								{statusComponents[1]}
+							</Badge>
+						)}
+					</TableCellLayout>
+				</TableCell>
+			</TableRow>
+		);
+	}
 
 	return (
-		<Stack className="container-summary-view">
-			<StackItem>
-				<span>
-					<b>Container</b>:{" "}
-					{containerState.nickname !== undefined
-						? `${containerState.nickname} (${containerState.id})`
-						: containerState.id}
-				</span>
-			</StackItem>
-			<StackItem>
-				<span>
-					<b>Status</b>: {statusString}
-				</span>
-			</StackItem>
-			<StackItem>
-				{containerState.clientId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Client ID</b>: {containerState.clientId}
-					</span>
-				)}
-			</StackItem>
+		<Stack>
 			<StackItem>
-				{containerState.audienceId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Audience ID</b>: {containerState.audienceId}
-					</span>
-				)}
+				<Table size="extra-small" ref={tableRef}>
+					{ContainerIDRow("Container", containerState.id)}
+					{ContainerStatusRow()}
+					{ContainerIDRow("Client ID", containerState.clientId)}
+					{ContainerIDRow("Audience ID", containerState.audienceId)}
```

Comment:
Nit: I think ContainerIDRow might want to be renamed GenericRow or something like that. When I first got to it above, I thought it was going to be specifically for the container id and nothing else.